spec/models/course/notification_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Course::Notification, type: :model do
```

Comment:
This has already been tested in activity_spec. What I thought is this may be duplicating.